nautobot/extras/tests/test_context_managers.py

```diff
@@ -56,6 +56,101 @@ def test_change_log_created(self):
         self.assertEqual(oc_list[0].changed_object, location)
         self.assertEqual(oc_list[0].action, ObjectChangeActionChoices.ACTION_CREATE)
 
+    def test_create_then_delete(self):
+        """Test that a create followed by a delete is logged as two changes"""
+        with web_request_context(self.user):
+            location_type = LocationType.objects.get(name="Campus")
+            location_status = Status.objects.get_for_model(Location).first()
```

Comment:
For consistency/readability with the following tests:

```suggestion
        location_type = LocationType.objects.get(name="Campus")
        location_status = Status.objects.get_for_model(Location).first()
        with web_request_context(self.user):
```